plugins/shared/loader/loader_test.go

```diff
@@ -55,8 +56,12 @@ func newHarness(t *testing.T, plugins []string) *harness {
 		t.Fatalf("failed to get self executable path: %v", err)
 	}
 
+	exeSuffix := ""
+	if runtime.GOOS == "windows" {
+		exeSuffix = ".exe"
```

Comment:
can we confirm that the LXC driver (first external plugin) builds with .ext extention on windows? cc @notnoop